google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/cloudpubsub/internal/AckSetTrackerImpl.java

```diff
@@ -36,13 +36,51 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class AckSetTrackerImpl extends TrivialProxyService implements AckSetTracker {
+  // Receipt represents an unacked message. It can be cleared, which will cause the ack to be
+  // ignored.
+  private static class Receipt implements Runnable {
```

Comment:
Runnable is the one interface you never need to implement explicitly, you can just use method references if you need a runnable. Just remove the interface, rename the method as "onAck", and return `receipt::onAck`

Reply:
Done